news/2022-02-07-beatmap-spotlights-season-5-winter-2022.md

```diff
@@ -0,0 +1,167 @@
+
+## Season schedule and leaderboards
+
+| Event | Time |
+| :-- | :-- |
+| Season Start | May 3rd, 2021 |
+| Timeshift Lobby: Playlist A | May 3rd - May 10th, 2021 |
+| Timeshift Lobby: Playlist B | May 10th - May 17th, 2021 |
+| Timeshift Lobby: Playlist C | May 17th - May 24th, 2021 |
+| Timeshift Lobby: Playlist D | May 24th - May 31st, 2021 |
+| INTERMISSION | May 31st - Jun 7th, 2021 |
+| Timeshift Lobby: Playlist A | Jun 7th - Jun 14th, 2021 |
+| Timeshift Lobby: Playlist B | Jun 14th - Jun 21st, 2021 |
+| Timeshift Lobby: Playlist C | Jun 21st - Jun 28th, 2021 |
+| Timeshift Lobby: Playlist D | Jun 28th - Jul 5th, 2021 |
+| Season Conclusion | Jul 5th, 2021 |
+
+<!-- adjust when the start date is known -->
+
+The Winter Season 2022 leaderboard will be available over at the [Beatmap Spotlights Season Winter 2022](/wiki/Beatmap_Spotlights/Seasons/2022_Winter) wiki article once the first batch of lobbies conclude in the coming weeks.
+
+## Rewards
+
+| Badges | Bracket Tier | Placement |
+| :--: | :--: | :--: |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/ri_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/ri_2.png) | Rhythm Incarnate | The best of the best |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/diamond_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/diamond_2.png) | Diamond | 3% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/platinum_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/platinum_2.png) | Platinum | 3% - 10% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/gold_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/gold_2.png) | Gold | 10% - 25% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/silver_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/silver_2.png) | Silver | 25% - 50% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/bronze_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/bronze_2.png) | Bronze | 50% - 70% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/copper_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/copper_2.png) | Copper | 70% - 95% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/iron_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/iron_2.png) | Iron | 95% - 100% |
+
+The percentage values may vary a little depending on the game mode and/or participant count.
+
+The Rhythm Incarnate tier threshold is picked manually based on season's participant count and general size of other tiers, though it is an absolute number that ranges between top 2 and top 50 in most cases.
+
+## <a id="playlists"></a> Spring Season 2021 first playlists
+
+### osu! (lead by [Nowaie](https://osu.ppy.sh/users/5428909))
+
+#### Playlist A (unorthodox rhythm)

```

Comment:
Only having (Unorthodox rhythm) for the osu playlist seems kinds odd. May be better to be consistent with the naming within the newspost